packages/cli/commands/project/migrateApp.js

```diff
@@ -185,10 +185,11 @@ exports.handler = async options => {
       text: i18n(`${i18nKey}.migrationStatus.failure`),
       failColor: 'white',
     });
-    logApiErrorInstance(
-      error.error || error,
-      new ApiErrorContext({ accountId })
-    );
+    if (error.errors) {
+      error.errors.map(logApiErrorInstance);
```

Comment:
Nit: Since we are ignoring the result we can probably use `forEach`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1098/commits/5bac1b437a2995c326ecc9b67d5f9794d9554c7a